src/resources/codemirror.ondemand.ts

```diff
@@ -0,0 +1,13 @@
+interface LoadedCodeMirror {
+  codeMirror: any;
+  codeMirrorCss: any;
+}
+
+let loaded: Promise<LoadedCodeMirror>;
+
+export const loadCodeMirror = async (): Promise<LoadedCodeMirror> => {
+  if (!loaded) {
+    loaded = import(/* webpackChunkName: "codemirror" */ "./codemirror");
```

Comment:
I know we've done it like this before, but thinking now if WebPack caches this import call ?